Sources/UtiliKit/Container/Transitioning/ContainerTransitionAnimator.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ContainerViewControllerTransitionAnimator.swift
+//  UtiliKit
+//
+//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+
+/// This class is internal to the framework. It is the internal default transition animator used by the ContainerViewController when it's delegate does not provide one.
+class ContainerTransitionAnimator: NSObject, UIViewControllerAnimatedTransitioning {
```

Comment:
I know the comment mentions it, but would it be better to add "Default" or "Standard" into the class name here to give additional context around its use?

Reply:
I can do this - just want to point out this class is internal to the library and so shouldn't be required for anyone but library contributors.